content/en/docs/faq/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "FAQ"
+description: "FAQ is designed to answer and summarize the questions our users most frequently ask about KubeSphere."
```

Comment:
->
FAQ is designed to answer and summarize the questions users ask most frequently about KubeSphere.